Cargo.toml

```diff
@@ -1,3 +1,6 @@
+[profile.release]
+opt-level = "s"
```

Comment:
I suggest we use a different profile for ESP32. As it is, this change would pessimize runtime for everyone else.

Reply:
I think it can also be done in the same profile as long as the target is specified like with other options.